MeshLib/MeshSearch/ElementSearch.h

```diff
@@ -34,18 +34,43 @@ class ElementSearch final
     /// @tparam PROPERTY_TYPE integral type of the property
     /// Different properties can be assigned to the elements of the mesh. These
     /// properties can be accessed by the name of the property. The method marks
-    /// all elements of the mesh for the property \c property_name with the
-    /// given property value \c property_value.
-    /// @param property_value the value of the property the elements have to
-    /// have to be marked
+    /// all elements of the mesh for the property \c property_name with  a
+    /// property value equal to \c property_value.
     /// @param property_name the name of the property the searching/marking is
     /// based on
+    /// @param property_value value required for the element to be marked
     /// @return The number of marked elements will be returned. The concrete
     /// element ids can be requested by getSearchedElementIDs().
     template <typename PROPERTY_TYPE>
-    std::size_t searchByPropertyValue(
-        PROPERTY_TYPE const property_value,
-        std::string const& property_name)
+    std::size_t searchByPropertyValue(std::string const& property_name,
+                                      PROPERTY_TYPE const property_value)
+    {
+        return searchByPropertyValueRange<PROPERTY_TYPE>(
+            property_name, property_value, property_value, false);
+    }
+
+    /// @tparam PROPERTY_TYPE integral type of the property
```

Comment:
I'm not sure I understand what you write here. What do you mean by "integral type"? I'm asking because in the above implementation you added `double` and `int`, where the former is not an "integral type".

Reply:
I have no idea, actually. I just copied the comment from the previous method, because the range-search basically replaces the (single-)value search. I have no idea who wrote that comment originally.

I've removed the word "integral", is that okay with you?

Reply:
:heavy_check_mark: